toast-tk-web-agent/src/main/java/io/toast/tk/agent/web/rest/AsyncHttpClientProvider.java

```diff
@@ -1,15 +1,28 @@
 package io.toast.tk.agent.web.rest;
 
-import com.google.common.base.Strings;
-import com.google.inject.Inject;
-import com.google.inject.Provider;
-import io.toast.tk.agent.config.AgentConfig;
-import org.asynchttpclient.AsyncHttpClientConfig;
+import java.security.cert.CertificateException;
+
+import javax.net.ssl.SSLException;
+
+import org.apache.http.conn.ssl.TrustSelfSignedStrategy;
+import org.apache.http.ssl.SSLContextBuilder;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.http.ssl.SSLContextBuilder'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)